Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -87,18 +87,10 @@ Country VolumeGC::GetCountry(const Partition& partition) const
   const u8 country = ReadSwapped<u8>(3, partition).value_or(0);
   const Region region = GetRegion();
 
-  // Korean GC releases use NTSC-J.
-  // E is normally used for America, but it's also used for English-language Korean GC releases.
-  // K is used by games that are in the Korean language.
-  // W means Taiwan for Wii games, but on the GC, it's used for English-language Korean releases.
-  // (There doesn't seem to be any pattern to which of E and W is used for Korean GC releases.)
```

Comment:
Should (parts of) this comment be moved to `CountryCodeToRegion`/`CountryCodeToCountry`? I could imagine that at least the `expected_region` ones would benefit from some distinction (since it isn't necessarily obvious from just the code).